app/src/main/res/values/third_party_dependencies.xml

```diff
@@ -21,91 +21,56 @@
     <item>@string/third_party_dependency_version_3</item>
   </string-array>
   <string name="license_text_0">"
```

Comment:
Here & elsewhere: remove quotes?

Also, should these licenses be in a CDATA to preserve formatting?

Reply:
https://developer.android.com/guide/topics/resources/string-resource.html#escaping_quotes
Quotes help in preserve text formatting. CDATA only helps for HTML text parsing it doesn't handle whitespaces.

Reply:
Ah I see. Thanks for clarifying.